### PR TITLE
fix: web3.min.js cdn 404

### PR DESCRIPTION
**Pull request by Pumpkir**
## Description

[https://docs.web3js.org/guides/web3_providers_guide/#injected-providers](https://docs.web3js.org/guides/web3_providers_guide/#injected-providers)

This web3.min.js cdn link is 404
photo_2023-10-08_00-10-07

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran `npm run lint` with success and extended the tests and types if necessary.
- [x] I ran `npm run test:unit` with success.
- [x] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [x] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [x] I have tested my code on the live network.
- [x] I have checked the Deploy Preview and it looks correct.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: docs/docs/guides/web3_providers_guide/index.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ As stated above, the injected provider should be in compliance with [EIP-1193](h
 The web3.js 4.x Provider specifications are defined in [web3 base provider](https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-types/src/web3_base_provider.ts) for Injected Providers.
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/web3@4.0.1-rc.1/dist/web3.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/web3@4.1.2/dist/web3.min.js"></script>
```
Comment (jdevcs) [inline docs/docs/guides/web3_providers_guide/index.md]: @Pumpkir Thanks, 
I think also add in documentation that replace this URL with latest release link listed in: https://www.jsdelivr.com/package/npm/web3

Comment (Muhammad-Altabba) [reply docs/docs/guides/web3_providers_guide/index.md]: > @Pumpkir Thanks, I think also add in documentation that replace this URL with latest release link listed in: https://www.jsdelivr.com/package/npm/web3

Good idea @jdevcs ,
I also found that you can get the last major version like this:
https://cdn.jsdelivr.net/npm/web3/dist/web3.min.js
And the last minor like this:
https://cdn.jsdelivr.net/npm/web3@4/dist/web3.min.js

So we can use one of the above and do not add in the documentation a note asking the reader to replace the Url. However, which one, of the above Url, do you recommend using?

Comment (jdevcs) [reply docs/docs/guides/web3_providers_guide/index.md]: https://cdn.jsdelivr.net/npm/web3/dist/web3.min.js is more good

Comment (jdevcs) [reply docs/docs/guides/web3_providers_guide/index.md]: @Pumpkir so can you replace all CDN URLs in doc with above one. thanks

Comment (Pumpkir) [reply docs/docs/guides/web3_providers_guide/index.md]: Of course, I found two and have completed the replacement

**Review by jdevcs — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Pull request merged**